cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/bindings/strategies/DefaultMissingInputStrategy.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.runtime.bindings.strategies;
+
+import io.cloudslang.lang.entities.bindings.Input;
+import io.cloudslang.lang.runtime.env.RunEnvironment;
+import io.cloudslang.lang.runtime.events.LanguageEventData;
+import io.cloudslang.score.lang.ExecutionRuntimeServices;
+import io.cloudslang.score.lang.SystemContext;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+/**
+ * Default strategy to keep existing behavior
+ */
+@Component
+public class DefaultMissingInputStrategy implements MissingInputStrategy {
```

Comment:
I think _MissingInputHandler_ is a better name instead of _MissingInputStrategy_

`DefaultMissingInputStrategy `can be renamed to `EnforceValueMissingInputHandler`